### DISCUSSION TITLE
Update rating of packages to reflect any documentation

### DISCUSSION THREAD

**Discussion by pydanny** [djangopackages/djangopackages#660] [Feature Ideas]
Currently only uses Read the Docs. Should include any viable alternative.

**Comment by philippeowagner** [comment]
:+1:

**Comment by ryancheley** [comment]
In looking at the code, I'm not seeing how scoring for documentation (Read the Docs or otherwise) is calculated. The `Package` model has a `calculate_score` method with the following reference

```
# penalty for docs maybe?
return max(-500, self.repo_watchers - last_updated_penalty - python_3_penalty)
```

I think this is worth pursuing, but potentially as part of a larger review / update of the scoring algorithm in general
